examples/CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ foreach(example ${example_directories})
       # See alternate approach in a2113e0997c9 if this becomes too slow
       BUILD_ALWAYS 1
       CMAKE_ARGS
-        "-DCMAKE_PREFIX_PATH=${FAKE_INSTALL_PREFIX};${example_INSTALL_DIR}"
```

Comment:
this seems find

the history of using two paths here goes back a long time (https://github.com/gazebosim/gz-cmake/commit/3ea5909342b2b66427c5ace44e43e0a4c05b0b77), so if CI is happy with just `FAKE_INSTALL_PREFIX`, then that should be fine